2.4/README.md

```diff
@@ -25,6 +25,9 @@ The following environment variables influence the MongoDB configuration file. Th
 |  `MONGODB_NOPREALLOC` | Disable data file preallocation.                                          |  true
 |  `MONGODB_SMALLFILES` | Set MongoDB to use a smaller default data file size.                      |  true
 |  `MONGODB_QUIET`      | Runs MongoDB in a quiet mode that attempts to limit the amount of output. |  true
+|  `MONGODB_TEXT_SEARCH_ENABLED` | Enables the [text search](https://docs.mongodb.org/v2.4/core/index-text/#text-search-text-command) feature.  |  false
+
+**Notice: The `MONGODB_TEXT_SEARCH_ENABLED` option is deprecated in MongoDB 2.6.**
```

Comment:
@bparees OK, I removed the notice saying this option is deprecated in 2.6, it will simply not exist there. 

I added the warning for not using this in production systems.